edge-middleware/bot-protection-datadome/pages/_app.tsx

```diff
@@ -13,14 +13,14 @@ function MyApp({ Component, pageProps }: AppProps) {
       title="Bot Protection with DataDome"
       path="edge-middleware/bot-protection-datadome"
       deployButton={{
-        env: ['NEXT_PUBLIC_DATADOME_CLIENT_KEY', 'DATADOME_SERVER_KEY'],
+        env: ['DATADOME_CLIENT_SIDE_KEY', 'DATADOME_SERVER_SIDE_KEY'],
       }}
     >
       <Component {...pageProps} />
 
       {/* datadome bot protection */}
       <Script strategy="lazyOnload" id="load-datadome">{`
-        window.ddjskey = '${process.env.NEXT_PUBLIC_DATADOME_CLIENT_KEY}'
+        window.ddjskey = '${process.env.DATADOME_CLIENT_SIDE_KEY}'
```

Comment:
Is this actually on the client side if it doesn't use `NEXT_PUBLIC_`? We might want to rename. Looks like it's used in server only stuff?